modules/google-maps/src/utils.ts

```diff
@@ -287,3 +310,9 @@ function handleMouseEvent(deck, type, event) {
       return;
   }
 }
+
```

Comment:
I'm really not sure whether we should export `assert` from [core](https://github.com/visgl/deck.gl/pull/6822/files#diff-c7af049053d7ecbaeda943f96b0bc615a658be293117ce9185afc87593742ef8R90) or redefine in each of module.